### PR TITLE
Changed release process to use a GitHub Actions workflow

### PR DESCRIPTION
**Pull request by andy-maier**

### CODE REVIEW

**Review by KSchopmeyer — COMMENTED** [review]
I noticed that this file does not include the item to endit the README file.  Is that just missing or is there a reason?

**Review by KSchopmeyer — APPROVED** [review]

**Comment by andy-maier** [comment]
Readme file: In the pywbem instructions, that is not the readme file of pywbem but of pywbem.io

File: docs/changes.rst [hunk 1]
```diff
@@ -28,6 +28,9 @@ Released: not yet
 
 **Enhancements:**
 
+* Development: Changed release process to use a GitHub Actions workflow.
+  (issue #1395)
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply docs/changes.rst]: done

**Comment by andy-maier** [comment]
I now also added the publish.yml workflow that had been forgotten in the first version of the PR.

**Review by KSchopmeyer — APPROVED** [review]

**Comment by KSchopmeyer** [comment]
Marked approved.

**Pull request merged**